finalPrep/modZeroHero.js

```diff
@@ -74,11 +113,42 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName(){
+    console.log(this.name);
+  }
+  maximizeEnergy(){
+    this.energyLevel = 1050;
+  }
+  gainPower(num){
```

Comment:
Interesting- this comes down to a developer decision, but I am not sure you'd also add the '100' to whatever the user inputs. I would think it more natural to just increase the powerLevel by whatever is input. Regardless, nice job making this dynamic function!